README.md

```diff
@@ -144,6 +144,30 @@ This SDK implements the [best practices for user authentication via an OpenID Co
 
 ## Simplified Login
 
+### Configuring Simplified Login
+Prerequisite: Applications need to be on the same Apple Development account in order to have access to the shared keychain. 
+
+1. In your application target, add Keychain Sharing capability with keychain group set to `com.schibsted.simplifiedLogin`.
+2. Creating client, pass additional parameter `appIdentifierPrefix`. It is usually the same as team identifier prefix - 10 characters combination of both numbers and letters assigned by Apple.
+```swift
+let client = Client(configuration: clientConfiguration, appIdentifierPrefix: "xxxxxxxxxx") 
+```
+3. Create `SimplifiedLoginManager` and call `getSimplifiedLogin` method anytime you want to present SL to user.
+```swift
+    let context = ASWebAuthSessionContextProvider()
+    let manager = SimplifiedLoginManager(client: client, contextProvider: context, env: clientConfiguration.env) { result in
+            print("Catch login result \(result)")
+    }
+    manager.getSimplifiedLogin("Your brand name visible on UI") { result in
+        switch (result) {
+        case .success():
+            print("success")
+        case .failure(let error):
+            print("Catch error from presenting SL \(error)")
+        }
+    }
+```
+
```

Comment:
I am going to update this readme after final version of PR https://github.com/schibsted/account-sdk-ios-web/pull/90